core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -168,6 +168,27 @@ private[spark] object TestUtils {
     createCompiledClass(className, destDir, sourceFile, classpathUrls)
   }
 
+  /** Create a dummy compile jar for a given package, classname.  Jar will be placed in destDir */
+  def createDummyJar(destDir: String, packageName: String, className: String): String = {
+    val srcDir = new File(destDir, packageName)
+    srcDir.mkdirs()
+    val excSource = new JavaSourceFromString(new File(srcDir, className).getAbsolutePath,
```

Comment:
Sure.  Whats the easiest way to get the R unit tests running locally on a windows box?